utils.js

```diff
@@ -4617,18 +4627,63 @@ export async function parseCargoTomlData(cargoData) {
           pkg._integrity = "sha384-" + value;
           break;
         case "name":
+          value = value.split(" ")[0];
           pkg.group = dirname(value);
           if (pkg.group === ".") {
             pkg.group = "";
           }
           pkg.name = basename(value);
           break;
         case "version":
-          pkg.version = value;
+          pkg.version = value.split(" ")[0];
+          break;
+        case "authors":
+          pkg.author = value.replace(/[[\]]/g, "");
+          break;
+        case "homepage":
+          pkg.homepage = { url: value };
+          break;
+        case "repository":
+          pkg.repository = { url: value };
+          break;
+        case "license":
+          pkg.license = value;
           break;
       }
```

Comment:
Because Cargo workspaces are not supported yet, do you think it would be a good idea to add a debug print here informing the user that the parent component object may be wrong if a `[workspace]` section is?

This is mostly for cases similar to [Rust's `Cargo.toml`-file](https://github.com/rust-lang/rust/blob/master/Cargo.toml) which does not have a `[package]`-section in the top-level file, so the implementation will use whichever `[package]` or `[dependencies]` section appears first. The user can get around this by invoking the tool for each sub-package directory separately but it would help the user to find that solution if they are informed.

Reply:
Interesting. Let me take a look.